pkg/controller/broker/v2alpha3/activemqartemis/activemqartemis_reconciler.go

```diff
@@ -1227,16 +1227,28 @@ func getDeployedResources(instance *brokerv2alpha3.ActiveMQArtemis, client clien
 	var log = logf.Log.WithName("controller_v2alpha3activemqartemis")
 
 	reader := read.New(client).WithNamespace(instance.Namespace).WithOwnerObject(instance)
-	resourceMap, err := reader.ListAll(
-		&corev1.PersistentVolumeClaimList{},
-		&corev1.ServiceList{},
-		&appsv1.StatefulSetList{},
-		&routev1.RouteList{},
-		&extv1b1.IngressList{},
-		&corev1.SecretList{},
-	)
+	var resourceMap map[reflect.Type][]resource.KubernetesResource
+	var err error
+	if isOpenshift, _ := environments.DetectOpenshift(); isOpenshift {
+		resourceMap, err = reader.ListAll(
+			&corev1.PersistentVolumeClaimList{},
+			&corev1.ServiceList{},
+			&appsv1.StatefulSetList{},
+			&routev1.RouteList{},
+			&extv1b1.IngressList{},
```

Comment:
I added `&extv1b1.IngressList{},` for Kubernetes, maybe after this split we could remove it from the OpenShift branch

Reply:
right, will remove that. Thanks!